sdk/communication/Azure.Communication.Chat/tests/samples/Sample2_MessagingOperations.cs

```diff
@@ -60,8 +60,9 @@ await foreach (ChatMessage message in allMessages)
             await chatThreadClient.UpdateMessageAsync(messageId, "updated message content");
             #endregion Snippet:Azure_Communication_Chat_Tests_Samples_UpdateMessage
 
+            //Note : Due to the async nature of the storage, moving coverage to CI tests
             #region Snippet:Azure_Communication_Chat_Tests_Samples_SendReadReceipt
-            await chatThreadClient.SendReadReceiptAsync(messageId);
+            //@@await chatThreadClient.SendReadReceiptAsync(messageId);
```

Comment:
I do not think this scenario actually makes a lot of sense. When a user sends a message, they do not need to say that they have read it themselves, do they? If not, considering the timing issues, I would say let's remove it till we have another way of testing it.

Reply:
No, this is just for the markup, and it shows how to send a read receipt for a given message (not necessarily the one that was just sent by the user). Maybe we can make it more clear somehow ? 
Also, the actual sending is not happing here (//@@ is an annotation to do not run that line). I already moved all the readreceipts actual coverage to its own test.

Reply:
We can have another sample that someone gets messages and mark them as read.

Reply:
Was looking at how the readme file looks like, and it actually looks ok to me. It probably does not show in the exact order in the code, but let me know you still think we need to change anything ?
![image](https://user-images.githubusercontent.com/2148028/95268834-f3ddae80-07ec-11eb-8f22-dce6e42fb997.png)